src/FakeItEasy/RootModule.cs

```diff
@@ -92,7 +92,7 @@ public static void RegisterDependencies(IServiceRegistrar registrar)
             registrar.Register<IFakeObjectCallFormatter>(fakeObjectCallFormatter);
 
             StringBuilderOutputWriter StringBuilderOutputWriterFactory() =>
-                new StringBuilderOutputWriter(argumentValueFormatter);
+                new StringBuilderOutputWriter(argumentValueFormatter!);

```

Comment:
Not sure why that was necessary. There's no way `argumentValueFormatter` is null at this point.

Reply:
Yeah. Ah well, it's a minor annoyance.